models/articles.model.js

```diff
@@ -0,0 +1,20 @@
+const db = require('../db/connection');
+
+exports.fetchArticleById = (id) => {
+    
+    if (isNaN(Number(id))){
```

Comment:
This isn't necessary and is a bit circular. You are checking if the id coerced into a number is not a number which can never be true as a Number will always be a number

Reply:
the error you refer to is when we pass a parameter that isn't a number and this triggers a psql error that we should handle in app :)

Reply:
think I've done this now